web_services/foxx/brightid/index.js

```diff
@@ -194,156 +196,91 @@ const handlers = {
     });
   },
 
-  allVerificationsGet: function(req, res){
+  verificationPublicGet: function(req, res){
     const appKey = req.param('app');
-    const count_only = 'count_only' in req.queryParams;
     const app = db.getApp(appKey);
-    const context = db.getContext(app.context);
-    const coll = arango._collection(context.collection);
-    let contextIds = db.getLastContextIds(coll, app._key);
-    let data = {
-      count: contextIds.length
-    }
-    if (! count_only){
-      data['contextIds'] = contextIds
+    const roundedTimestamp = req.param('roundedTimestamp');
+    const verification = req.param('verification') || app.verification;
```

Comment:
The `info` object may change in the future.  Especially `roundedTimestamp` may be optional or handled differently.  Suppose that a certain app decides to not use timestamps, but to have a custom info element called `votingRound` with a uuid value.

I think it's best to take an `info` object and use it as-is.

Reply:
If something is not required to be verified by the node, why should that be added to `info`?
I think `info` should include things that node should verify and sign.

Reply:
`info` can contain things that the node doesn't touch, but that make it a unique signature.  Those things are included so the node can know whether or not to sign it (because it hasn't seen that `info` before for that user).

Look at my example of `votingRound` with a unique, unguessable value.

Anyway, if we go the same route of the smart contracts for all verifications (i.e. nodes always verify the blind signatures), then the apps get the verification status by contextid from the nodes, and we can limit what can be in `info` to whatever we choose as sensible defaults.  This is simplicity over flexibility.  In this case `app`, `verification`, and `roundedTimestamp` are ok.

If that's how we want to handle it, then it's better to construct the `info` object--like you're doing--so that a user can't get two signatures by leaving off the `verification` parameter to get the default value once, and then including it another time.

Reply:
> `info` can contain things that the node doesn't touch, but that make it a unique signature. Those things are included so the node can know whether or not to sign it (because it hasn't seen that `info` before for that user).

Technically apps can not add anything to info based on current implementation, because blind signatures are created in advance by the client as soon as users got apps' required verifications, before user has any interaction with apps.